R/teal_data-constructor.R

```diff
@@ -19,22 +19,22 @@
 #' @details
 #'
 #' A `teal_data` is meant to be used for reproducibility purposes. The class inherits from
-#' [`teal.data::qenv`] and we encourage to get familiar with \CRANpkg{teal.code} first.
+#' [`teal.code::qenv`] and we encourage to get familiar with \CRANpkg{teal.code} first.
 #' `teal_data` has following characteristics:
 #'
 #' - It inherits from the environment and methods such as [`$`], [get()], [ls()], [as.list()],
 #' [parent.env()] work out of the box.
 #' - `teal_data` is a locked environment, and data modification is only possible through the
-#' [teal.code::eval_code()] and [within.qenv()] functions.
-#' - It stores metadata about the code used to create the data (see [get_code()]).
+#' [teal.code::eval_code()] and [teal.code::within.qenv()] functions.
+#' - It stores metadata about the code used to create the data (see [get_code,teal_data-method]).
```

Comment:
Doesn't look good
<img width="545" alt="Skärmavbild 2025-01-28 kl  08 49 53" src="https://github.com/user-attachments/assets/458a55ec-297e-4b88-aabf-27a1ea9ce092" />



```suggestion
#' - It stores metadata about the code used to create the data (see [get_code()][get_code,teal_data-method]).
```